openslides_backend/action/actions/motion_workflow/create.py

```diff
@@ -21,10 +22,76 @@ class MotionWorkflowCreateAction(CreateActionWithDependencies):
     dependencies = [MotionStateActionSet.get_action("create")]
 
     def get_dependent_action_payload(
-        self, instance: Dict[str, Any], CreateActionClass: Type[Action]
+        self, instance: Dict[str, Any], CreateActionClass: Type[Action], index: int
     ) -> Dict[str, Any]:
         return {
             "name": MOTION_STATE_DEFAULT_NAME,
             "workflow_id": instance["id"],
             "first_state_of_workflow_id": instance["id"],
         }
+
+
+@register_action("motion_workflow.create_simple_workflow")
+class MotionWorkflowCreateSimpleWorkflowAction(CreateActionWithDependencies):
+    """
+    Action to create a motion workflow.
+    """
+
+    model = MotionWorkflow()
+    schema = DefaultSchema(MotionWorkflow()).get_create_schema(
+        ["name", "meeting_id"],
+        [
+            "default_workflow_meeting_id",
+            "default_amendment_workflow_meeting_id",
+            "default_statute_amendment_workflow_meeting_id",
+            "first_state_id",
+        ],
+    )
+    dependencies = [MotionStateActionSet.get_action("create")] * 4
+
+    def get_dependent_action_payload(
+        self, instance: Dict[str, Any], CreateActionClass: Type[Action], index: int
+    ) -> Dict[str, Any]:
+        next_state_ids: List[int] = []
+        if index == 3:
+            for wr in self.write_requests:
+                for event in wr.events:  # type: ignore
+                    event_fields = cast(Dict[str, Any], event["fields"])
+                    if (
+                        event["fqid"].collection.collection == "motion_state"
+                        and event["type"] == EventType.Create
+                        and event_fields["workflow_id"] == instance["id"]
+                    ):
+                        next_state_ids.append(event_fields["id"])
+                        self.additional_relation_models[event["fqid"]] = event_fields
```

Comment:
Same as above, manually searching through the events to find an id doesn't seem like the right approach. With `execute_other_action` this becomes a lot cleaner (you don't have to use `CreateActionWithDependencies` to use it).

Reply:
Done